Add a way to whitelist tweets

sw00 opened the pull request:
Sometimes you want to keep a tweet or two, e.g. pinned tweets.

vickylai commented:
Excellent, except that IMO the `WHITELIST` variable should be optional.

Running the function with no `WHITELIST` variable set in Lambda produces: 

```
panic: missing required environment variable WHITELIST
```

sw00 commented:
Sorry, dropped the ball on that.

`whitelist` now defaults to empty/zero-sized slice when not defined. Added a function just for it - can't overload `getenv` because return type is `[]string` instead of `string`.
